webpack-dev-server.config.js

```diff
@@ -1,11 +1,13 @@
 /* @flow */
 /* eslint unicorn/filename-case: 0 */
-import { BASE, getWebpackConfig } from './webpack.config';
```

Comment:
Did you mean to change this file too?

Reply:
Yeah, fixing linter errors to get the build to pass. Was originally in its own PR but, cant get the build to pass without these other changes :)

Reply:
Aha I had some fixes locally for this -- I've pushed them up

Reply:
Got it, I just removed those changed from this branch